homeassistant/components/velbus/cover.py

```diff
@@ -51,7 +51,9 @@ def __init__(self, channel: VelbusChannel) -> None:
     @property
     def is_closed(self) -> bool | None:
         """Return if the cover is closed."""
-        return self._channel.is_closed()
+        if val := self._channel.is_closed():
```

Comment:
What are the possible values of `val` here?

Reply:
None (in case we don't know yet), True or False (in case we know)

Reply:
You could just `return self._channel.is_closed()` then